hack/local-up-cluster.sh

```diff
@@ -417,6 +423,14 @@ function start_apiserver {
     if [[ -n "${NODE_ADMISSION}" ]]; then
       security_admission=",NodeRestriction"
     fi
+    if [ "${ENABLE_POD_PP}" == true ]; then
```

Comment:
maybe some var that is less cryptic? POD_PRIORITY_AND_PREEMPTION ?

Reply:
Thanks. BTW, should  var  prefix with `ENABLE`? I think `ENABLE_POD_PRIORITY_PREEMPTION` better?

Reply:
Looks like there is no rule here in the script. ENABLE_ is fine with me.